src/main/java/teammates/common/util/Const.java

```diff
@@ -942,6 +942,7 @@
         public static final String FEEDBACK_RESULTS_NOTHINGNEW = "You have not received any new feedback but you may review your own submissions below.";
         
         public static final String ENROLL_LINE_EMPTY = "Please input at least one student detail.";
+        public static final String FEEDBACK_SESSION_RESULTSSECTIONVIEWWARNING = "This session seems to have a large number of responses. It is recommended to view the results one section at at time.";
```

Comment:
FEEDBACK_SESSION_RESULTSSECTIONVIEWWARNING can be changed to  FEEDBACK_SESSION_RESULTS_SECTIONVIEWWARNING, to be consistent with similar messages, and moved up one line so the variables are sorted.